### PR TITLE
Nitesh branch

### PR DESCRIPTION
**Pull request by IAMNITESHPANDIT**

### CODE REVIEW

**Comment by IAMNITESHPANDIT** [comment]
Kindly approve it .

File: frontend/src/App.js [hunk 1]
```diff
@@ -8,52 +8,46 @@ import "./App.css";
 class App extends Component {
```
Comment (keshavvyastechvalens) [inline frontend/src/App.js:8]: Change it into functional Component

Comment (keshavvyastechvalens) [inline frontend/src/App.js:8]: Convert to functional component

Comment (IAMNITESHPANDIT) [reply frontend/src/App.js:8]: Already Fixed 👍

**Review by keshavvyastechvalens — APPROVED** [review]
Only App.js is class component

**Pull request merged**